utils.ts

```diff
@@ -241,3 +237,38 @@ export const mergeResponses = (
 
   return mergedResponse
 }
+
+/**
+ * Makes a geocoder request by first checking a potential redis store for a cached
+ * response. If a response is fetched, stores response in redis store.
+ * @param redisClient     Redis client already connected
+ * @param requestMethod   Geocoder Request Method
+ * @param args            Args for Geocoder request method
```

Comment:
Reorder and add `@param` for geocoder

Reply:
Thank you for catching this one! d8598b93ef1d8cfc2e66faffd38be81c9a9e442c